coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -0,0 +1,301 @@
+      </v-card>
+    </v-dialog>
+
+    <!-- Transition to Payment -->
+    <!-- TODO - this should be on Payment page -->
+    <v-fade-transition>
+      <div class="loading-container" v-show="showLoading">
+        <div class="loading__content">
+          <v-progress-circular color="primary" :size="50" indeterminate></v-progress-circular>
+          <div class="loading-msg">{{this.loadingMsg}}</div>
+        </div>
+      </div>
+    </v-fade-transition>
+
+    <div id="standalone-directors" ref="standaloneDirectors">
+      <!-- Initial Page Load Transition -->
+      <div class="loading-container fade-out">
+        <div class="loading__content">
+          <v-progress-circular color="primary" :size="50" indeterminate></v-progress-circular>
+          <div class="loading-msg">Preparing Your Filing</div>
+        </div>
+      </div>
+
+      <v-container id="standalone-directors-container" class="view-container">
+        <article id="standalone-directors-article">
+          <header>
+            <h1 id="AR-header">Directors - As of {{ lastFilingDate }}</h1>
+          </header>
+
+          <div>
+            <!-- Director Information -->
+            <section>
+              <header>
+                <h2 id="AR-step-3-header">xxx</h2>
+              </header>
+                <Directors @directorsChange="directorsChangeEventHandler" ref="directorsList" :asOfDate="currentDate" />
+            </section>
+          </div>
+        </article>
+
+        <aside>
+          <affix relative-element-selector="#standalone-directors-article" :offset="{ top: 120, bottom: 40 }">
+            <sbc-fee-summary v-bind:filingData="[...filingData]" v-bind:payURL="payAPIURL"/>
+          </affix>
+        </aside>
+      </v-container>
+
+      <v-container id="submit-container" class="pt-0">
+        <div class="filing-buttons">
+          <v-btn
+            id="cod-pay-btn"
+            color="primary"
+            large
+            @click="submit">
+            File &amp; Pay
+          </v-btn>
+          <v-btn
+            id="cod-cancel-btn"
+            large
+            to="/dashboard">
+            Cancel
+          </v-btn>
+        </div>
+      </v-container>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import axios from '@/axios-auth'
+import Directors from '@/components/AnnualReport/Directors.vue'
+import { Affix } from 'vue-affix'
+import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
+import { mapState, mapActions, mapGetters } from 'vuex'
+
+export default {
+  name: 'StandaloneDirectorsFiling',
+
+  components: {
+    Directors,
+    SbcFeeSummary,
+    Affix
+  },
+
+  data () {
+    return {
+      showLoading: false,
+      loadingMsg: 'Redirecting to PayBC to Process Your Payment',
+      directorsChange: false,
+      filingData: [],
+      dialog: false,
+      lastFilingDate: 'something' // TODO
+    }
+  },
+
+  computed: {
+    ...mapState(['noAGM', 'regOffAddrChange',
```

Comment:
Looks like the following aren't used: noAGM, regOffAddrChange, validated, currentARStatus. Not sure why we'd need something like addressesFormValid and directorFormValid, so I assume this is still in progress?